release-notes/aws/10.1.0.md

```diff
@@ -0,0 +1,40 @@
+## :zap:  Giant Swarm Release 10.1.0 for AWS is now active for you! :zap:
+
+We are pleased to announce that this new branch (10.1.0) contains the much anticipated Node Pools release and will contain all
+other releases thereafter such as High Availability of Masters and Spot Instances which are due to be released next year. 
+
+We therefore kindly remind you that from this release and for a period of six months onwards, 
+to give you a chance to move your clusters, we will be supporting both this new branch and the 'Legacy' 9.0.0 branch which we will
+continue to maintain during this period. 
+
+In the meantime, we hope you enjoy using Node Pools of which the documentation can be found here: https://docs.giantswarm.io/basics/nodepools/ 
+including the other release changes which are listed below: 
+
+### Kubernetes v1.15.5
+- Updated from v1.14.6 - [changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.15.md#kubernetes-v115-release-notes)
```

Comment:
The latest release 9.0.0 is 1.15.5 already. So imo there is no upgrade here.

Reply:
So can I just delete this?